lib/markbind/lib/parser.js

```diff
@@ -376,8 +376,8 @@ Parser.prototype.includeFile = function (file, config) {
       const userDefinedVariables = config.userDefinedVariablesMap[path.resolve(parent, relative)];
       const fileContent = nunjucks.renderString(data, userDefinedVariables);
       const fileExt = utils.getExtName(file);
-      if (fileExt === 'md') {
-        context.source = 'md';
+      if (utils.isMarkdownFileExt(fileExt)) {
+        context.source = fileExt;
```

Comment:
Do we need `context.source` to be the actual `fileExt`? Seems cleaner if we can treat custom Markdown file extensions as `'md'`.